src/components/Garden.js

```diff
@@ -0,0 +1,143 @@
+import React, { useEffect, useRef } from 'react';
+import * as d3 from "d3";
+import './Garden.css';
+
+const Garden = (props) => {
+
+  const colorsByLanguage = {
+    "JavaScript": '#DE2016',
+    "HTML": '#FF3EAA',
+    "CSS": '#FCD732',
+    "SCSS":'#26369E',
+    "Python":'#76678C',
+    "Java":'#F19233'
```

Comment:
we should replace these colors with more color scheme (planned based on common stacks as you suggested)